api/observability/v1/clusterlogforwarder_types.go

```diff
@@ -0,0 +1,190 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/status"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
+type ClusterLogForwarderSpec struct {
```

Comment:
According to the enhancement proposal the spec should include a `CollectorSpec` (See [here](https://github.com/openshift/enhancements/pull/1537/files#diff-9a031ef6be7961e9c14a506fda2e573134aca811685c5a0707b2e878cdc2afcbR133-R138). Mistake or missing for good?

Reply:
Missing.  It crossed my mind during impl but was inadvertently left off in the flurry to push something on Friday